### ISSUE TITLE
Provide an apt repository

### ISSUE THREAD

**Issue by lathiat** [crmarsh/discord-linux-bugs#68]
Please provide an apt repository.  This will allow the package to be automatically updated with the standard system tools.

This is critical for providing timely and effortless security updates, not only because that's a good idea generally but the client is using chromium/webkit behind the scenes rendering public user input -- a notorious target for security issues making it a doubly good idea.

It's not particularly difficult to do, if you need some help or tips on how to do it then I can offer suggestions.  One point I would make is that you can install the repository with the package itself, making for easy '1 click' desktop installation and then providing ongoing updates in a single step.  Which is friendlier than inputting various terminal commands.  Google Chrome uses this approach, if you wish to analyse another party doing that.

**Comment by shaneshort** [comment]
+1

**Comment by bluntelk** [comment]
This would be awesome

**Comment by Serkan-devel** [comment]
+1

**Comment by TheDoctorsLife** [comment]
This might be a good idea for now, however they are trying to move away from having to update the actual application, and are moving more towards just updating the web version instead. However there is the need to keep electron and its stuff up to date with the latest security and bug patches, therefore doing something like this is still a good idea to go through with.

**Comment by crmarsh** [comment]
Yeah, I will spend some more time on getting updates right before we release a stable build.

**Comment by NaGeL182** [comment]
+1

**Comment by tkdberger** [comment]
Please don't comment '+1', there's a button to add you reaction to a post/comment in the top right of its respective box. It looks like a plus sign with a smiley face, clicking it will let you add a simple emoji reaction to a post without cluttering the place with useless +1 comments.

**Comment by GreyAsteroid** [comment]
+1
